app/views/journalist/articles/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<p>
+    <strong>Title:</strong>
+    <%= @article.title %>
+</p>
+
+<p>
+    <strong>Lead:</strong>
+    <%= @article.lead %>
+</p>
+ 
+<p>
+    <strong>Text:</strong>
+    <%= @article.content %>
+</p>
+<%= link_to "Edit", edit_journalist_article_path %>
```

Comment:
should all journalists be able to edit and delete all articles?

Reply:
thats a later user story

Reply:
If you look at my previous comment with the user story, the journalist should only be able to edit / delete his own articles.

Reply:
done